src/pyhf/infer/mle.py

```diff
@@ -34,8 +36,13 @@ def fit(data, pdf, init_pars=None, par_bounds=None, **kwargs):
         ... )
         >>> observations = [51, 48]
         >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
-        >>> pyhf.infer.mle.fit(data, model, return_fitted_val=True)
-        (array([0.        , 1.0030512 , 0.96266961]), array([24.98393521]))
+        >>> bestfit_pars, twice_nll = pyhf.infer.mle.fit(data, model, return_fitted_val=True)
+        >>> bestfit_pars
+        array([0.        , 1.0030512 , 0.96266961])
+        >>> twice_nll
+        array([24.98393521])
+        >>> -2 * model.logpdf(bestfit_pars, data) == twice_nll
```

Comment:
Or should we show this with 
```suggestion
        >>> pyhf.infer.mle.twice_nll(bestfit_pars, data, model) == twice_nll
```
? Not sure which makes things more clear.